agilerl/wrappers/pettingzoo_wrappers.py

```diff
@@ -39,7 +37,7 @@ def step(self, actions: dict[AgentID, ActionType]) -> tuple[
         dict[AgentID, dict],
     ]:
         obs, rewards, terminations, truncations, infos = self.env.step(actions)
-        if np.any(list(terminations.values())) or np.any(list(truncations.values())):
+        if np.all(list(terminations.values()) or list(truncations.values())):
```

Comment:
I am still unsure about tying the reset condition to all agents' termination. Did PettingZoo developers provide feedback on this? @nicku-a @mikepratt1

Reply:
Hey Burak, we believe this is the correct behaviour as there are certain Petting Zoo environments where individual agents may terminate but the episode can continue (we observed this behaviour in [Knights, Archers, and Zombies](https://pettingzoo.farama.org/environments/butterfly/knights_archers_zombies/) ).  If you want your environment to reset with this previous method, we suggest either adjusting your environment or adding a simple wrapper to it to return termination = True for all agents if one agent terminates.